spec/support/validator_spec.rb

```diff
@@ -1,23 +1,23 @@
-RSpec.shared_examples "validate valid parameter" do |validator_class, parameter|
+RSpec.shared_examples 'validate valid parameter' do |validator_class, parameter|
   context "with parameter #{parameter}" do
-    subject(:valid_parameter_validator) { validator_class.new('name', validator_definition, parameter).tap {|validator| validator.validate} }
+    subject(:validator) { validator_class.new('name', definition, parameter).tap {|validator| validator.validate} }
 
     it 'is valid' do
-      expect(valid_parameter_validator.is_valid).to be_truthy
+      expect(validator.is_valid).to be_truthy
     end
   end
 end
 
-RSpec.shared_examples "validate invalid parameter" do |validator_class, parameter, errored_parameters|
+RSpec.shared_examples 'validate invalid parameter' do |validator_class, parameter, errors|
   context "with parameter #{parameter}" do
-    subject(:invalid_parameter_validator) { validator_class.new(name, validator_definition, parameter).tap {|validator| validator.validate} }
+    subject(:validator) { validator_class.new(name, definition, parameter).tap {|validator| validator.validate} }
```

Comment:
I named these subject's differently because there was a namespace clash, but then I put them in a seperate context so that probably isn't a problem anymore